lib/JitEngineDevice.hpp

```diff
@@ -39,6 +40,31 @@
 // TODO: check if this global is needed.
 static llvm::codegen::RegisterCodeGenFlags CFG;
 
+class JITKernelEntry {
+  char const* Name;
+  int32_t* RCIndices;
+  int32_t NumRCs;
+public:
+  JITKernelEntry(char const* _Name,
+                 int32_t* _RCIndices,
+                 int32_t _NumRCs) :
+                 Name(_Name),
+                 NumRCs(_NumRCs) {
+                  // TODO(bowen): Why is this deep copy needed?
```

Comment:
Remove comment, you need to copy because those are allocated in the stack through the ProteusPass instrumentation